index.html

```diff
@@ -6,8 +6,150 @@
+            with a lovely view while watching a beautiful sunset Lorem ipsum,
+            dolor sit amet consectetur adipisicing elit. Natus, doloremque enim!
+            Necessitatibus fuga voluptas voluptatibus. Quia, laboriosam deleniti
+            omnis nam aliquam libero accusantium dolorum dolor corrupti placeat
+            tempore, nesciunt debitis.
+          </p>
+          <p class="para2">28.10.2022 (FRIDAY)</p>
+          <p class="para3">
+            Catch them live at the Botanic Gardens on the above date
+          </p>
+        </div>
+      </main>
+    </header>
+    <section id="events" class="events">
+      <h3 class="upcoming-events">Up Coming Events</h3>
+      <hr class="breakline" />
+      <div class="events-list">
+        <div class="event1">
+          <img src="./images/program_icon_01.png" alt="" />
+          <h4>Magic Day Club</h4>
+          <p>Mea Culpa, 11th Road, South Africa 30 OCT 2022 13:00 ZAR250</p>
+        </div>
+        <div class="event1">
+          <img src="./images/program_icon_02.png" alt="" />
+          <h4>Music at Zevenwacht</h4>
+          <p>Zevenwacht Wine Estate, Cape Town FROM 30 OCT 2022 12:00 ZAR250</p>
+        </div>
+        <div class="event1">
+          <img src="./images/program_icon_03.png" alt="" />
+          <h4>OktoberFest</h4>
+          <p>The Sanctuary Kloof Street 30 OCT 2022 12:00 ZAR350</p>
+        </div>
+        <div class="event1">
+          <img src="./images/program_icon_04.png" alt="" />
+          <h4>Berg's Boys Choir</h4>
+          <p>KwaZulu Natal, Durban ICC 2 November 2022 ZAR 160</p>
+        </div>
+        <div class="event1">
+          <img src="./images/program_icon_05.png" alt="" />
+          <h4>Gqebe Music Awards</h4>
+          <p>Boardwalk Casino Hotel 30 October 2022 ZAR200</p>
+        </div>
+      </div>
+      <div class="book-events">
+        <h5>BOOK EARLY 2022</h5>
+      </div>
+    </section>
+    <section class="featured-artists">
+      <h3 class="feature-h3">Featuring Artists</h3>
+      <hr class="breakline" />
+      <div class="artists-list">
+      </div>
+      <button>
+        MORE
+        <img src="./images/down.png" alt="Down arrow" />
+      </button>
+    </section>
+    <section>
+      <div class="partners">
+        <h2>Partners</h2>
+        <hr class="breakline" />
+        <div class="partners-img">
+          <div>
+            <img src="./images/mozilla.png" alt="Mozilla brand" />
+          </div>
+          <div>
+            <img src="./images/google.png" alt="Google brand" />
+          </div>
+          <div>
+            <img src="./images/naver.png" alt="Naver brand" />
+          </div>
+          <div>
+            <img src="./images/kakao.png" alt="Kakao brand" />
+          </div>
+          <div>
+            <img src="./images/airbnb.png" alt="Airbnb brand" />
+          </div>
+        </div>
+      </div>
+      <div class="about-footer">
```

Comment:
- [ ] It would be semantic if you used a `<footer></footer>` tag for the footer instead of a div 👍